src/components/Camera/index.js

```diff
@@ -61,7 +62,8 @@ class CameraError extends React.Component<CameraErrorType> {
   errorInstructions = (text) =>
     <span onClick={this.onFallbackClick} className={style.errorLink}>
       { text }
-      <input type="file" accept='image/*' capture
+      <input type="file" accept='image/*'
+        capture={ this.props.method === 'face' ? 'user' : true }
```

Comment:
I don't think you need `method` here. We are only displaying this error on face camera.

Reply:
Thanks @stefaniacardenas 

`Camera` will handle any errors coming from `react-webcam` and pass a `CAMERA_NOT_WORKING` error to `CameraPure`, since we are at the moment rendering all errors with tags using the `errorInstruction()` method, that error will render an upload fallback:

```
"camera_not_working": {
  "instruction":  "It may be disconnected or not functional. <uploadFallback>Try again.</uploadFallback>"
}
```

In this case, I think we need to know what the user is trying to do before setting the value of `capture`